base/grisu.jl

```diff
@@ -13,9 +13,12 @@ include("grisu/float.jl")
 include("grisu/fastshortest.jl")
 include("grisu/fastprecision.jl")
 include("grisu/fastfixed.jl")
+include("grisu/bignums.jl")
 include("grisu/bignum.jl")
 
-function grisu(v::FloatingPoint,mode,requested_digits,buffer=DIGITS)
+const BIGNUMS = [Bignums.Bignum(),Bignums.Bignum(),Bignums.Bignum(),Bignums.Bignum()]
```

Comment:
Adding another "module global" here isn't ideal I know as we're heading towards mult-threading, but it added another ~10% performance boost, so I figured it was worth it (and we can solve it when we solve the `BUFFER` variable as well). I can take it out however, if we don't want to dig deeper in that direction.